test/parallel/test-fs-promises-file-handle-write.js

```diff
@@ -35,6 +35,17 @@ async function validateEmptyWrite() {
   assert.deepStrictEqual(buffer, readFileData);
 }
 
+async function validateNonUint8ArrayWrite() {
+  const filePathForHandle = path.resolve(tmpDir, 'tmp-empty-write.txt');
+  const fileHandle = await open(filePathForHandle, 'w+');
+  const buffer = Buffer.from('Hello world', 'utf8').toString('base64');
+
+  await fileHandle.write(buffer, 0, buffer.length);
+  const readFileData = fs.readFileSync(filePathForHandle);
+  assert.deepStrictEqual(Buffer.from(buffer, 'utf8'), readFileData);
+}
+
 validateWrite()
   .then(validateEmptyWrite)
+  .then(validateNonUint8ArrayWrite)
   .then(common.mustCall());
```

Comment:
It would be great to run those validations in parallel instead of waiting on the result of the former ones. Therefore it would be good to write this as:

```js
Promise.all([
  validateWrite(),
  validateEmptyWrite(),
  validateNonUint8ArrayWrite()
]).then(common.mustCall());
```

To make sure that works as expected, the temp files have to be unique though (have a unique name).

Reply:
@BridgeAR : Thanks for the feedback. As you said, makes sense to run those validations in parallel. I've updated the PR with the changes. Kindly review. Thanks.